### ISSUE TITLE
Monthly Product Update in Intranet

### ISSUE THREAD

**Issue by gianfranco-l** [giantswarm/roadmap#1628]
## Status quo

- Team monthly Product update written in Slackbot and sent out to news-product.

## Proposal

- Monthly Product update written down in the Intranet + TL;DR and link to the relevant Intranet page sent out via Slackbot to news-product;
- Where in Intranet? Current suggestion: subsection under [Product](https://intranet.giantswarm.io/docs/product/);
- Structure:
     - 1 specific page per update;
     - TBD: 1 section per team / 1 section per month / other ?

cc @giantswarm/sig-product @giantswarm/product-owners

**Comment by marians** [comment]
@gianfranco-l Let me knoe if you want assistance with creating some example content.

BTW I don't think we have to copy the entire history from Slack into the intranet. An example set of a few pages should do.

**Comment by gianfranco-l** [comment]
@marians right, the rationale is to document from the moment we use the new proposal, not the past :) 
regarding the structure of this new section,this is yet to be defined AFAIK :) we'd need some direction here from the team that owns the intranet (if exists :) )

**Comment by puja108** [comment]
To keep it simple, I'd say we just start with `Product / Updates / TeamName` as folders and then each update is a page/.md with format `YYYY-MM.md` this way the autosorting should also work.

We can iterate on the structure if we see people having issues with it. 

Content structure I'd also start with keeping what we have and just adding a TLDR at the top. @marians helping out with some nice examples might be cool.

**Comment by gawertm** [comment]
@puja108 I was jsut thinking if we divide by team, one would have to click through each team to see the latest updates. not sure how convenient this will be. but I also do not have a better approach in mind

**Comment by puja108** [comment]
Right the alternative would then be going with `Product / Updates` and then `YYYY-MM-teamname.md`, could still work well.

**Comment by marians** [comment]
+1 for a flatter structure. Should be doable.

**Comment by marians** [comment]
Some general content questions.

Currently geekbot asks for past and future.

- What did your team achieve last month?
- What will you do this month?

Do we want to keep it this way?

Since the updates are usually not written at the end/beginniong of a month, the understanding of "last month" and "this month" is a bit fluent, I guess. Do we see it as something like a moving 30 days window?

The reason I'm asking is that it's simple and clean two use headlines like `Team X – October 2022`, but if the post covers a bit of September and a bit of October, this headline format makes little sense.

**Comment by marians** [comment]
## Intranet proposal

Using the blog layout provided by docsy.

`/docs/product/bulletin/`

![localhost_1313_docs_product_bulletin_](https://user-images.githubusercontent.com/273727/202719108-d4e74e53-a75b-4b7b-9377-a27ea0ef6239.png)

`/docs/product/bulletin/2022-11-08-atlas/`

![localhost_1313_docs_product_bulletin_2022-11-08-atlas_](https://user-images.githubusercontent.com/273727/202718665-e8675f79-7b3d-4e04-b8d7-e2512813cbd8.png)

`/docs/product/bulletin/2022-11-15-rocket/`

![localhost_1313_docs_product_bulletin_2022-11-15-rocket_ (1)](https://user-images.githubusercontent.com/273727/202718961-374783f1-b6a5-4382-b4c7-6b1144fc4760.png)

**Comment by marians** [comment]
To check it out live, please use https://github.com/giantswarm/giantswarm/pull/24673

**Comment by gawertm** [comment]
@marians @puja108 I was even thinking about an even more flat structure like 1 .md file per months with all the teams in it. Or would it get too big?
and I also think we should stick roughly to a calendar months e.g. product update october, november, etc.

**Comment by marians** [comment]
In SIG Product sync I just got the feedback that the part about plans is a duplication if what POs tend to cover in the public roadmap issues. So I'll remove that from the draft.

I will also work towards supporting full content display on the list page. (The requirement of having a distinct URL for each news posting has been mentioned, so there will still be a details link for each one.)

**Comment by gawertm** [comment]
hi, we were running out of time in the meeting, but I actually disagree little with what @alex-dabija was saying about the plan being a duplicate to the roadmap. I think in the monthly product updates you would summarize it in a different way. Also you would include private things, collaboration topics, etc. that are not in the public roadmap.

**Comment by marians** [comment]
## Latest preview

Here is now a proposal where all content is shown on the index page. Also the Acknowledgement vs. Plans distinction is removed from the example content.

`/docs/product/bulletin/`

![localhost_1313_docs_product_bulletin_ (1)](https://user-images.githubusercontent.com/273727/203299765-8f3083e1-40e2-4fdc-936a-1f4f5365d5c4.png)

`/docs/product/bulletin/2022-11-08-atlas/`

![localhost_1313_docs_product_bulletin_2022-11-15-rocket_ (2)](https://user-images.githubusercontent.com/273727/203299784-738181da-18d8-44c6-8341-746772020062.png)

`/docs/product/bulletin/2022-11-15-rocket/`

![localhost_1313_docs_product_bulletin_2022-11-08-atlas_ (1)](https://user-images.githubusercontent.com/273727/203299805-efe01724-0131-4a11-8ea0-37a441af460a.png)

**Comment by gawertm** [comment]
can we have this instruction in intranet also?

**Comment by marians** [comment]
@gawertm Sure. I just don't. find a good place where to put it. Any idea?

**Comment by gawertm** [comment]
I also dont know. nothing really fits. maybe just under the bulletin? How to monthly product update. or in the documentation section..

**Comment by marians** [comment]
The howto is now placed in the intranet, under Product / Bulletin.

@gianfranco-l @puja108 Do you want to keep the issue around, or close it?

**Comment by puja108** [comment]
Sorry, only now catching up on this. I'll communicate and adapt the geekbot

**Comment by puja108** [comment]
I adapted geekbot with a reminder to do the PR and then the question just asks you to post your TLDR with a link.

On that note, I also added in the reminder that we want a TLDR that is posted to Slack (but also included in the MD) as we discussed early on.

Also, rethinking our discussion on Roadmap being the plans for next months, we do indeed sometimes have internal and customer related updates that would be good to keep internal as @gawertm mentioned. @giantswarm/product-owners WDYT, I'd say esepcailly for now until we get the Roadmap board in order to actually be able to serve as a source of truth, let's keep the updates. We can re-evaluate once the Roadmap board is regularly used and maintained and we see people actually using it. I worry not just about having the information, but also about communication of said information, clickthroughrates from Slack might actually be quite low and some of that information is quite important.

**Comment by gianfranco-l** [comment]
I am very confused because the intended objective of this proposal was to have a single source of truth of product team updates, the end result seems to have 3-4 places (TLDR in slack, bulletin in intranet, roadmap board and availability overview) were to share some part of info or duplicate one. Besides unclear where to put which information for whom and why, I think the end result would lead more confusion to stakeholders and more effort on teams rather than make everything easier.

**Comment by marians** [comment]
I agree with Gianfranco in that it's getting more complicated. I suggest to skip the demand to posting a duplication of the bulletin to Slack. (If people want to post and duplicate, I also would not stop them. But it has to be clear that there is an intranet URL for this sort of info.)

Regarding future plans per team: Does it get simpler if we say "Only mention items that are not covered by the Roadmap"?

**Comment by gianfranco-l** [comment]
to better refine my statement [here](https://github.com/giantswarm/roadmap/issues/1628#issuecomment-1331809199), it would actually be 6 places: the aforementioned 4 + the roadmap Miro board + changelog of sales availability.

**Comment by gawertm** [comment]
so what are you suggesting?

**Comment by puja108** [comment]
To clarify, the roadmap board, availability overview and roadmap miro (incl changelog) are not things that have been added with this change. They were iterations that were made within SIG product over the last few months. As mentioned this monday in SIG product I agree that some aggregation can and should be done to reduce the organizational effort for POs. Horizon will work on some improvements (e.g. status syncing between team and roadmap board) here and we believe that the roadmap board can be an essential tool for communication for us also internally, with which we can get rid of some other boards (e.g. sig product board) an communication tools. I believe that it can replace the current miro with the availability overview and the simplified roadmap in miro, but for that we need to put some work into actually using it. In it's current form it is not a good replacement for the info we have abstracted elsewhere.

I can get on board with the idea though to take a step back in the coming year and refocus on the problems we are trying to solve on the communication side and find fitting solutions for them.

For now I'd close this issue, and refer to Horizon issues regarding roadmap board iterations.
